Resources/Prototypes/Damage/modifier_sets.yml

```diff
@@ -384,3 +384,10 @@
   id: PotassiumIodide
   coefficients:
     Radiation: 0.1
+
+- type: damageModifierSet 
+  id: Tajaran
+  coefficients: # Literally the same as felinids, but seperate cuz there's no parenting?
```

Comment:
No, parenting doesn't affect the choice of damage modifier, your prototypes do. See IPCs, which were overpowered because they used Metallic just like walls.